.github/workflows/benchmark.yaml

```
@@ -48,12 +48,15 @@ jobs:
 
       - name: integration tests
         run: |
-          go test -bench Benchmark ./integration/... | tee benchmark.txt
+          go test -bench Benchmark ./... | tee benchmark.txt
```

I'd suggest using this instead:
```suggestion
          go test -run='^$' -bench=. ./... | tee benchmark.txt
```